README.md

```diff
@@ -24,30 +34,33 @@ see https://neo4j.com/docs/operations-manual/current/backup/
 
 When working localy, docker-compose.override.yml adds a local bind mount to the /backups folder.
 
+The Crunchbase backup can be downloaded from Nexus. Decompress it in the ./backups bind mounted folder.
```

Comment:
here it's any dataset, do you think it is worth reflecting this ?

Reply:
That's right. I'll change it.